omnisafe/evaluator.py

```diff
@@ -261,6 +272,12 @@ def __load_model_and_env(
             )
 
         else:
+            if 'Saute' in self._cfgs['algo'] or 'Simmer' in self._cfgs['algo']:
+                observation_space = Box(
+                    low=-np.inf,
+                    high=np.inf,
+                    shape=(observation_space.shape[0] + 1,),
+                )
```

Comment:
this may change original observation space limitation. recommand using such as
```python
np.hstack((observation_space.high, np.inf))
```
to get the bound. and pass an array to `high` and `low`. You can change all other same conditions as this also.